server.py

```diff
@@ -30,6 +32,10 @@
 from app.utility.config_generator import ensure_local_config
 
 
+celery_app = Celery('caldera', include=['app.utility.tasks'])
```

Comment:
So celery is not toggable when booting Caldera, it will be enabled every time correct?

Reply:
I'm a bit hesitant about making this jump to completely remove the previous option if we can help it. Using celery is definitely a step up, but it also increases the complexity of everything - though I'm not sure if we can support two systems in parallel.